tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -3035,6 +3036,29 @@ int BPF_KPROBE(trace_security_socket_create)
     return events_perf_submit(&data, SECURITY_SOCKET_CREATE, 0);
 }
 
+SEC("kprobe/security_path_symlink")
```

Comment:
security_path_* is guarded with CONFIG_SECURITY_PATH:
https://elixir.bootlin.com/linux/latest/source/include/linux/security.h#L1767
This config is not enabled on some distros, so we better not use it, and definitely not as a default event, which will fail tracee to start.

security_inode_symlink can be used instead

Reply:
> Fixed - There is one downside now though: 2 events are generated when a 'ln -s' is happening inside a container. One with the paths relevant for the container, and the other for the overlay fs. Unlike for when hooking security_path_symlink, which happens only in the container scope.

There might be an easy fix for this.
By looking at inode->i_sb->s_magic and comparing it to overlayfs magic number, you can know if the path is within the container or not